[Website] Show opening hours

Philipp000 opened the pull request:
Resolves #114 

Ready for merge after #105 

Displays the opening hours for a canteen on the gh-pages website.
This looks currently as follows:
![Bildschirmfoto 2022-01-25 um 19 59 44](https://user-images.githubusercontent.com/1690395/151042949-61d14f84-f859-46f6-b96e-7430aa7b4149.png)

The text can have different colors:
- red, when the canteen already closed
- green, when the canteen is open
- blue, when the canteen will open today
- black, when the selected day is not today

Further, a click on the info icon will open a modal, that shows all opening hours for the given canteen:
![Bildschirmfoto 2022-01-25 um 19 59 51](https://user-images.githubusercontent.com/1690395/151043217-96c14e70-ff3b-49f9-aa35-d0c9fbc3c60e.png)

Philipp000 commented:
Additionally, I now added information, on which text color means what in the modal:
![Bildschirmfoto 2022-01-26 um 16 46 30](https://user-images.githubusercontent.com/1690395/151196696-05dc864e-5ba6-48fa-a177-dbfccf5731f9.png)

Further, I refactored the code of the modals, to not have duplicate code.

Philipp000 commented:
As the opening hours are now merged, this should now function properly

COM8 commented:
![image](https://user-images.githubusercontent.com/11741404/151360137-218ef4b9-c00d-4fb2-9073-6eec1de3705f.png)
A small margin between the opening hours and the menu would be great here.

![image](https://user-images.githubusercontent.com/11741404/151360253-31073574-9bac-460e-b9fc-cdb73a5ae194.png)
Also the hline at the end of the table is missing and is the white background "blur" here intended?

Philipp000 commented:
I have now added a margin and the missing end of the table.

However, I'm not sure on what you refer to with the "blur"?
You mean, that the background in general is white? Actually, I think having a dark background for the modal even in dark mode makes more sense.
I already created a PR a while ago in the library for that, however it is not merged yet: https://github.com/jloh/bulma-prefers-dark/pull/72

If you meant this by the white blur, I can also implement it, as it looks there.

Philipp000 commented:
I have now updated the styling of the modal, the PR should now be ready for merging.

I have updated:
- Darker background color for modal in dark mode
- Move close button into the modal
It now looks like this:
![Bildschirmfoto 2022-01-28 um 19 21 20](https://user-images.githubusercontent.com/1690395/151600948-be4c5587-9022-42fb-b25b-c9742d8a07eb.png)

COM8 reviewed (approved):
Looks good. Thanks!
